src/controllers/itemDetails/index.js

```diff
@@ -439,8 +439,20 @@ function renderName(item, container, context) {
     if (parentNameHtml.length) {
         if (parentNameLast) {
             // Music
+            // Determine if there are over 10 artists in the album. If so, concat with 'and X others.'
+            const artists = parentNameHtml[0].split(' / ');
+            const otherArtistCount = artists.length - 10;
+            let newParentNameHtml = artists.slice(0, 10).join(' / ');
```

Comment:
This approach of parsing the generated html doesn't seem like a great approach... I think it would be best to handle this in the `getArtistLinksHtml` function:

https://github.com/jellyfin/jellyfin-web/blob/b7f341ce087d1e5a4448121bfb7a743717b2e1ef/src/controllers/itemDetails/index.js#L385-L398

Reply:
Yeah I would agree with that. I will update the code.

Reply:
Code updated in latest push.